config/locales/en.yml

```diff
@@ -837,6 +837,7 @@ en:
             success: Files were successfully concatenated.
           modal:
             title: Concatenate Files
+            description: "The following files are going to be concatenated:"
```

Comment:
Let's change this text to `The following files were selected to be concatenated:`

Reply:
a8c391e95c0036aa40376d35bd1eaf43d4acf8f9